9c-internal/9c-pluggable/application.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: 9c-pluggable
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/planetarium/9c-infra.git
+    targetRevision: 9c-pluggable
+    path: charts/9c-network
+    helm:
+      valueFiles:
+        - ./values.yaml
```

Comment:
this path is the relative path from the chart. the chart is located in `charts/9c-network`. so value file path should be like `../../9c-internal/9c-pluggable/values.yaml`